apps/gitness/src/pages/repo/repo-commits.tsx

```diff
@@ -28,6 +28,7 @@ export default function RepoCommitsPage() {
     repo_ref: repoRef,
     queryParams: { page: 0, limit: 10, git_ref: normalizeGitRef(selectedBranch) }
   })
+  console.log(commitData)
```

Comment:
Do you need debugg this data in the next branch?

Reply:
fixed